presto-product-tests/bin/run_on_docker.sh

```diff
@@ -102,7 +102,7 @@ function stop_docker_compose_containers() {
     stop_application_runner_containers ${ENVIRONMENT}
 
     # stop containers started with "up", removing their volumes
-    environment_compose down -v
+    environment_compose down -v || true
```

Comment:
Then we should have a comment next to that line to make it clear it shouldn't be that way but we cannot fix it.